test/jdk/jdk/incubator/vector/Double128VectorTests.java

```diff
@@ -60,6 +60,8 @@ public class Double128VectorTests extends AbstractVectorTest {
     static final int INVOC_COUNT = Integer.getInteger("jdk.incubator.vector.test.loop-iterations", 100);
 
 
+    // for floating point reduction ops that may introduce rounding errors
+    private static final double RELATIVE_ROUNDING_ERROR = (double)0.01;
```

Comment:
It seems a bit large. Have you tried to make it smaller? Or what is your justification for this value?

Reply:
I'm just worried that 1% is very much for the addition tests. Basically we might be dropping off a whole element, and would not notice it.